### ISSUE TITLE
Segfaults with alpha

### ISSUE THREAD

**Issue by ankane** [bab2min/tomotopy#77]
Hi @bab2min, just wanted to report some segfaults I came across when building the Ruby library.

```python
import tomotopy as tp

model = tp.GDMRModel()
print(model.alpha)

model = tp.DTModel()
print(model.alpha)
```

tomotopy version: 0.9.1

**Comment by bab2min** [comment]
It has been identified as an error caused by accessing uninitialized data. It will be fixed at the next update.
Thank you for your reporting!

**Comment by bab2min** [comment]
fixed at version 0.10.0

**Comment by ankane** [comment]
Thanks @bab2min, works great!
